website/README.md

```diff
@@ -1,13 +1,37 @@
-# BuildBuddy's docs/blog website
+# buildbuddy.io marketing website
 
-To run the docs/blog website locally, run:
+## Development
+
+To run the marketing website locally, run:
+
+```bash
+bazel run website:start
+```
+
+View the marketing site at http://localhost:3000/
+
+This page will live reload as you make and save changes.
+
+# Production preview
+
+To preview a static production build locally, run:
+
+```bash
+bazel run website:serve
+```
+
+View the marketing site at http://localhost:3000/
+
+You'll need to rebuild this target for it to pick up any changes.
```

Comment:
could you add a line about what the difference between these is?

Reply:
Done